lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -45,6 +48,45 @@ class DataReductionFITSWriter(Tool):
         file_ok=True,
     ).tag(config=True)
 
+    event_filters = traits.Dict(
+        help="Enter the event filters for standard parameters - "
+        "intensity, leakage_intensity_width_2, r, wl",
+        default_value=dict(
+            {
+                "intensity": [100, np.inf],
+                "r": [0, 1],
+                "wl": [0.1, 1],
+                "leakage_intensity_width_2": [0, 0.2],
+            }
+        ),
+    ).tag(config=True)
+
+    fixed_cuts = traits.Dict(
+        help="Enter the fixed selection cut values for "
+        "gh_score(gammaness), theta and source_fov_offset",
+        default_value=dict(
+            {
+                "gh_score": 0.6,
+                "theta_cut": 0.2,
+                "source_fov_offset": 2.83,
+            }
+        ),
+    ).tag(config=True)
+
+    alpha = traits.Float(
+        help="Enter the selection cut for source dependent parameter - alpha",
+        default_value=8.0,
+    ).tag(config=True)
+
+    tel_ids = traits.Dict(
```

Comment:
Why is tel_ids a dict and not simply a list of ints?

Reply:
I just combined the two telescopes to make it a dict to make it more general. I can as well make it in 2 traitlets.